### DISCUSSION TITLE
D.3.3. Multilingual Support - EKG/Maturity

### DISCUSSION THREAD

**Discussion by giscus** [EKGF/ekg-maturity#84] [Page Comments]
# D.3.3. Multilingual Support - EKG/Maturity

Maturity Model for the Enterprise Knowledge Graph (EKG)

https://maturity.ekgf.org/pillar/technology/user-interface/multilingual-support/

**Comment by jgeluk** [comment]
This is just one detail of making everything "context-specific" and "localized"
